pinery-lims-miso/src/main/java/ca/on/oicr/pinery/lims/miso/MisoClient.java

```diff
@@ -1126,7 +1126,7 @@ public String extractStringValueFrom(ResultSet rs) throws SQLException {
           return null;
         }
       },
-      TARGETED_RESEQUENCING("targeted_resequencing", "Targeted Resequencing"), 
+      TARGETED_RESEQUENCING("targeted_sequencing", "Targeted Sequencing"), 
```

Comment:
Must keep the original attributeKey (second parameter) to match GSLE attribute name